authentication/identity_providers/configuring-github-identity-provider.adoc

```diff
@@ -6,30 +6,21 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Configure a `github` identity provider to validate user names and passwords
```

Comment:
Are there any modifications to the text beside long lines?

Reply:
Here, same as previous wondering comment https://github.com/openshift/openshift-docs/pull/39594/files#r1029248528 , for here, even more lines : )